components/datadog/agent/helm.go

```diff
@@ -197,6 +197,9 @@ func buildLinuxHelmValues(installName, agentImagePath, agentImageTag, clusterAge
 			"prometheusScrape": pulumi.Map{
 				"enabled": pulumi.Bool(true),
 			},
+			"kubelet": pulumi.Map{
+				"tlsVerify": pulumi.Bool(false),
+			},
```

Comment:
can we have this setting optionnel? 
IMO we should disable `tlsVerify` only if the k8s destribution doesn't have have a proper certificat, but by default consider that the certificat is valid.
Also it seems that for AKS this setting is not anymore needed see: https://docs.datadoghq.com/containers/kubernetes/distributions?tab=helm#aks-kubelet-certificate

Reply:
I agree that `kubelet.tlsVerify: false` shouldn’t be the default.
It should be needed only for Kind.

Reply:
Thank you for the info. I found a way to add environment-specific fields in the helm chart. It's done in this commit: 60cdcefbccd5e45c555fcf82c508456661f8a041.